tests/integration_tests/test_gpio/test_gpio.py

```diff
@@ -4,6 +4,10 @@
 import pytest
 from edgepi.gpio.edgepi_gpio import EdgePiGPIO
 
+def reset_pins(gpio, pin_name):
+    """Reset pins to power up default setting"""
+    gpio.set_pin_direction_in(pin_name)
+    gpio.set_pin_state(pin_name)
```

Comment:
Look at this: https://stackoverflow.com/questions/63813830/reset-class-and-class-variables-for-each-test-in-python-via-pytest

I think you can do set up/tear down in pytest rather than resetting after every test.

Reply:
the tear down method wasn't done because the `edgepi_gpio` module doesn't have a reset function. I wanted to reset the individual pins that are changed during the test. I actually see the needs of reset the pin now. I will open a ticket for this